### PR TITLE
Possibility of canceling pending or running jobs

### LINKED ISSUE #100: Cancel simulation being in progress

### PR DESCRIPTION
**Pull request by p1003**

### CODE REVIEW

File: yaptide/celery/tasks.py [hunk 1]
```diff
@@ -246,9 +248,38 @@ def get_input_files(job_id: str) -> dict:
...
+            "message": "Job canceled",
+            "was_canceled": True
+        }
+    return {
+        "message": "Something went wrong",
+        "was_canceled": False
+    }
 
 
 def translate_celery_state_naming(job_state: str) -> str:
```
Comment (grzanka) [inline yaptide/celery/tasks.py:285]: Add a test for this method

**Comment by grzanka** [comment]
@p1003 could you resume work on that ? It would be nice to at least cancel celery jobs without obtaining results. This PR needs desperately rebasing / merging master

**Comment by p1003** [comment]
> @p1003 could you resume work on that ? It would be nice to at least cancel celery jobs without obtaining results. This PR needs desperately rebasing / merging master

The changes in the project are so advanced that I think it would be better to just throw this PR to waste bin and rework it again from scratch. Do you agree?

**Comment by grzanka** [comment]
> > @p1003 could you resume work on that ? It would be nice to at least cancel celery jobs without obtaining results. This PR needs desperately rebasing / merging master
> 
> The changes in the project are so advanced that I think it would be better to just throw this PR to waste bin and rework it again from scratch. Do you agree?

Sure, we can safely close it

**Pull request closed without merging**